src/main/java/baritone/cache/CachedRegion.java

```diff
@@ -95,9 +95,7 @@ public final boolean isCached(int x, int z) {
                 if (locs == null) {
                     continue;
                 }
-                for (BlockPos pos : locs) {
-                    res.add(pos);
-                }
+                res.addAll(locs);
```

Comment:
This is a key inner loop for minebehavior, this is actually faster to iterate over the linked list one at a time, as opposed to addAll which converts to a [] internally.